examples/applications/restapi/handler_utils.py

```diff
@@ -0,0 +1,425 @@
+import asyncio
+import os
+from typing import Optional, Union, List, Dict, Any, Iterable, Tuple
+import bitcoinx
+from bitcoinx import TxOutput
+from aiohttp import web
+from munch import Munch
```

Comment:
Before you say anything... I don't intend to keep this dependency - I will force push over this commit when I complete the final touch ups. I am looking at a way to replace what it does currently - perhaps with attrs (https://www.attrs.org/en/stable/examples.html#converting-to-collections-types). Which we already have as a dependency.